pkgs/development/libraries/openexr/default.nix

```diff
@@ -4,10 +4,24 @@
 , fetchFromGitHub
 , zlib
 , ilmbase
-, fetchpatch 
+, fetchpatch
 , cmake
 , libtool
 }:
+
+let
+  non_glibc_fpstate_patch =
+    # Fix ilmbase/openexr using glibc-only fpstate.
+    # Found via https://git.alpinelinux.org/aports/tree/community/openexr/10-musl-_fpstate.patch?id=80d9611b7b8e406a554c6f511137e03ff26acbae,
+    # TODO Remove when https://github.com/AcademySoftwareFoundation/openexr/pull/798 is merged and available.
```

Comment:
This looks to be merged now.

Reply:
It's not `available` though (no release made).